src/Opdex.Platform.WebApi/Mappers/PlatformWebApiMapperProfile.cs

```diff
@@ -389,20 +389,32 @@ public PlatformWebApiMapperProfile()
             .ForMember(dest => dest.PledgeAmount, opt => opt.MapFrom(src => src.PledgeAmount))
             .ForMember(dest => dest.Approved, opt => opt.MapFrom(src => src.Approved));
 
+        CreateMap<VaultProposalsDto, VaultProposalsResponseModel>()
+            .ForMember(dest => dest.Results, opt => opt.MapFrom(src => src.Proposals))
+            .ForMember(dest => dest.Paging, opt => opt.MapFrom(src => src.Cursor));
+
         CreateMap<VaultProposalPledgeDto, VaultProposalPledgeResponseModel>()
```

Comment:
THink we're missing singular mappings here for all of these new models. 

`VaultProposalDto => VaultProposalResponseModel`
`VaultProposalPledgeDto => VaultProposalPledgeResponseModel`
`VaultProposalVoteDto => VaultProposalVoteResponseModel`

Reply:
These exist already

Reply:
ok 😅